artcommon/artcommonlib/rhcos.py

```
@@ -131,13 +131,12 @@ def get_build_id_from_rhcos_pullspec(pullspec, registry_config: str = None) -> s
     return build_id
 
 
-def get_latest_layered_rhcos_build(container_conf: Model, arch: str, major: int, registry_config: str = None):
+def get_latest_layered_rhcos_build(container_conf: Model, arch: str, registry_config: str = None):
```

Ruff (0.16.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)